### ISSUE TITLE
Outlook with UIA: page numbers are reported

### ISSUE THREAD

**Issue by leonardder** [nvaccess/nvda#9190]
cc @michaeldcurran 

### Steps to reproduce:
Open a message in the Outlook inbox with report pages enabled.

### Actual behavior:
NVDA announces the page number:

> <subject>
> dialog
> document
> page 1
> <First line>

### Expected behavior:
NVDA does not announce the page number.

I haven't looked at the code yet, but I recall that there's some logic to have page number reporting disabled in Outlook.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
alpha-16587

#### Windows version:
10 1809

#### Name and version of other software in use when reproducing the issue:
Outlook 16.0.11126.20266

#### Does the issue still occur after restarting your PC?
Yes

#### Have you tried any other versions of NVDA?
No
